wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/remote/CustomSocketLsLauncherProvider.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
```

Comment:
@JPinkney do we really need this? Maybe I am missing smth, but reading ws config and grabbing servers of a special type has been already implemented by @dkuleshov https://github.com/eclipse/che/blob/master/wsagent/che-core-api-languageserver/src/main/java/org/eclipse/che/api/languageserver/remote/SocketLsLauncherProvider.java Can't we just reuse it?